### DISCUSSION TITLE
The objects in searchParams do not support sting[]

### DISCUSSION THREAD

**Discussion by xiaozhen123321** [sindresorhus/ky#525] [Q&A] [answered]
I used get request and found that it cannot be written like this
get(url, {{searchParams: {a: 'a', b: ['c', 'd']}})
it's unreasonable
How to deal with this parameter

thanks

**Comment by sindresorhus** [comment, marked as answer]
The `searchParams` option is designed to follow the `URLSearchParams` API, which doesn't natively support array values.

**Comment by wtLau** [comment]
I would imagine you could build the search param using the `URLSearchParams` as the lib is designed with that API. 

So you could initialize using the `URLSearchParams` object instance, something like this:
```
const searchParams = new URLSearchParams({a: 'a', b: ['c', 'd']});

get(url, {{searchParams })
```
